ui/cypress/integration/standards/list.js

```diff
@@ -46,9 +49,15 @@ describe('Standards Listing Index', () => {
       it('Displays org matches first', () => {
         cy.visit('/standards');
         cy.get('input[name="q"]').type('prsb');
-
         cy.contains('Search').click();
-        cy.get('#browse-results li a').eq(0).click();
+
+        cy.get('#resultSummary')
+          .invoke('attr', 'data-loading')
+          .should('eq', 'false');
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(500);
```

Comment:
Is this still required?

Reply:
Sadly yes, I'm not sure _why_ but I'm also sure I'm at the end of being able to care about cypress edge cases on dom waits :(